doc/src/neighbor.rst

```diff
@@ -55,14 +55,24 @@ For the *bin* style, the bin size is set to 1/2 of
 the largest cutoff distance between any pair of atom types and a
 single set of bins is defined to search over for all atom types.  This
 can be inefficient if one pair of types has a very long cutoff, but
-other type pairs have a much shorter cutoff.  For style *multi* the
-bin size is set to 1/2 of the shortest cutoff distance and multiple
-sets of bins are defined to search over for different atom types.
+other type pairs have a much shorter cutoff. The *multi* style uses
+different sized bins for collections of different sized particles. Different
+sets of bins are then used to construct the neighbor lists as as further 
+described by Shire, Hanley, and Stratford :ref:`(Shire) <bytype-Shire>`. 
 This imposes some extra setup overhead, but the searches themselves
-may be much faster for the short-cutoff cases.
-See the :doc:`comm_modify mode multi <comm_modify>` command for a
-communication option that may also be beneficial for simulations of
-this kind.
+may be much faster. By default, separate collections of particles are defined 
+for each atom type. For systems with two or more types with similar 
+cutoffs, one can reduce the extra overhead by defining custom collections 
+using the :doc:`neigh_modify <neigh_modify>` command. See the 
+:doc:`comm_modify mode bytype <comm_modify>` command for compatible 
+communication options that may be beneficial for simulations of this kind.
+
+An alternate style, *multi/old*, sets the bin size to 1/2 of the shortest 
+cutoff distance and multiple sets of bins are defined to search over for 
+different atom types. This algorithm used to be the default *multi* 
+algorithm in LAMMPS but was found to be significantly slower than the new
+approach. Although, there may be instances where the *multi/old* style 
```

Comment:
For now we are keeping the old option in case there are use cases where multi/old outperforms the new multi style.